msteams-platform/toolkit/add-single-sign-on.md

```diff
@@ -68,28 +69,29 @@ The following steps help you to enable SSO in your application:
 > These changes are based on the templates we scaffold.
 
 ---
+
 <br>
 <br><details>
 <summary><b>Tab project
 </b></summary>
 
-1. Copy `auth-start.html` and `auth-end.htm`** in `auth/public` folder to `tabs/public/`. Teams Toolkit registers these two endpoints in Azure AD for Azure AD's redirect flow.
+1. Copy `auth-start.html` and `auth-end.htm`\*\* in `auth/public` folder to `tabs/public/`. Teams Toolkit registers these two endpoints in Azure AD for Azure AD's redirect flow.
 
 2. Copy `sso` folder under `auth/tab` to `tabs/src/sso/`.
 
-    * `InitTeamsFx`: The file implements a function that initializes TeamsFx SDK and opens `GetUserProfile` component after SDK is initialized
+   - `InitTeamsFx`: The file implements a function that initializes TeamsFx SDK and opens `GetUserProfile` component after SDK is initialized
 
-    * `GetUserProfile`: The file implements a function that calls Microsoft Graph API to get user info
+   - `GetUserProfile`: The file implements a function that calls Microsoft Graph API to get user info
 
 3. Execute `npm install @microsoft/teamsfx-react` under `tabs/`.
 
 4. Add the following lines to `tabs/src/components/sample/Welcome.tsx` to import `InitTeamsFx`:
```

Comment:
As per guidelines to write a folder path: In command syntax, use path to represent only the folder portion of the full path: copy [drive:][path]filename.

Please check and update the file path.

Reply:
As discussed with the PM, the PM explained that we can't add a dive path as it depends on where the user installs the package. He also confirmed that the folder structure remains the same.

Change not required.